tests/bats/helpers.bash

```diff
@@ -2,7 +2,12 @@ setup() {
   export VAULT_ADDR=http://127.0.0.1:8200
   export VAULT_TOKEN=root
   export VAULT_AUTHTYPE=token
-  docker run -d --net=host --name="vault-dev-server" --cap-add=IPC_LOCK -e 'VAULT_DEV_ROOT_TOKEN_ID=root' -p 8200:8200 -v /tmp/:/var/log/vault/:Z quay.io/app-sre/vault:1.5.4
+  docker run -d --name="vault-dev-server" \
```

Comment:
when you add the cleanup flow controls to the bash script, can you eliminate this setup() method entirely?

Reply:
also you can remove teardown below as well